src/components/general-components/MapCard.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
Is `general-components` a new directory? I don't think there is a need to have this subdirectory. Let's structure as follows: 
`src/components/mapCard/` directory
name this file `index.tsx`
name styles `mapCard.styles.less`
Then, you can import like `import MapCard from '../components/mapCard'`

Reply:
We do not call the react files index.tsx anywhere else, Makes more sense to me if the files themselves have descriptive names. 

However, the file structure makes sense and is probably less confusing for now, depending on how many components we have we may want to consider some way of organizing them.